config/mysql/master/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+rm -rf /etc/mysql/conf.d/*.cnf
+cp /tmp/mwdd/master.cnf /etc/mysql/conf.d/master.cnf
```

Comment:
I don't understand where that file is coming from ...

Reply:
/tmp/mwdd/master.cnf ?

It's coming from the mount:
"- ./config/mysql/slave:/tmp/mwdd"